docs/models.yml

```diff
@@ -1505,56 +1516,76 @@ motion_poll:
     equal_fields: meeting_id
   meeting_id:
     type: relation
-    to: meeting/motion_poll_ids
+    to: meeting/poll_ids
 
-motion_option:
+option:
   id: number
+  weight:
+    type: number
+    default: 10000
+  text: HTMLStrict
   yes: decimal(6)
   no: decimal(6)
   abstain: decimal(6)
 
   poll_id:
     type: relation
-    to: motion_poll/option_ids
+    to: poll/option_ids
     equal_fields: meeting_id
+    required: true
   vote_ids:
     type: relation-list
-    to: motion_vote/option_id
+    to: vote/option_id
     on_delete: CASCADE
     equal_fields: meeting_id
+  content_object_id:
+    type: generic-relation
+    to:
+      collections:  # Now, we have multiple models to vote about.
+        - collection: motion
+          field: option_ids
+        - collection: topic
+          field: option_ids
+        - collection: user
+          field:
+            name: option_$_ids
+            type: structured-relation
+            replacement: meeting_id
+    equal_fields: meeting_id
```

Comment:
Ok, this has to be implemented in the backend first before we can merge this...

(this is a new format, right? Or were structured generic relations possible before in the models code? Because you did not change the docs above)

Reply:
Yes this is new. Before this, you have one field decribing the field for all collections - now you can give a field descriptor for each collection.

Reply:
Then please add this to the docs